config.go

```diff
@@ -846,6 +850,11 @@ func loadConfig(appName string) (*config, []string, error) {
 		return nil, nil, err
 	}
 
+	// Enforce the minimum utxo cache max size.
+	if cfg.UtxoCacheMaxSize < minUtxoCacheMaxSize {
```

Comment:
I think there should probably be a sane upper limit too.  Since the amount of physical ram in the system is not easy to get (without an additional dependency we really don't want), I think 32768 (32 GiB) is more than sufficient as an upper limit.  A cache that size should be more than enough for the foreseeable future even with a massive increase in block size and number of utxos.  It could always be raise in future versions if needed since it's just a config option.

Reply:
Agreed that an upper limit is probably a good idea.  Added an upper limit of 32 GiB as suggested.